tests/e2e/cucumber/steps/ui/a11.y.ts

```diff
@@ -0,0 +1,11 @@
+import { Then } from '@cucumber/cucumber'
+import { World } from '../../environment'
+import { checkAccessibility } from '../../../support/utils/accessibility'
+
+Then(
+  '{string} check accessibility of the {string} page',
```

Comment:
I would suggest to change it to
`'{string} checks the accessibility of the DOM element {string} on the current page',`